_rules/image-no-text-0va7u6.md

```diff
@@ -0,0 +1,219 @@
+---
+id: 0va7u6
+name: HTML graphics contain no text
+rule_type: atomic
+description: |
+  This rule checks that images of text are not used
+accessibility_requirements:
+  wcag20:1.4.5: # Images of Text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.4.9: # Images of Text (No Exception) (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Carlos Duarte
+  images:
+    - Letter posted from Dulverton (Somerset, England) to Bristol (England) in 1894. Released into the public domain by Adrian Pingstone.
+    - Times Square image released into the public domain by (WT-shared) Ypsilonatshared at wts wikivoyage.
+---
```

Comment:
Should the words cloud have an entry here (not sure if you copied it or built it yourself…)

Reply:
I built it myself with a random words generator and word cloud generator :-)